modules/etcd-node-scaling.adoc

```diff
@@ -10,6 +10,11 @@ In general, clusters must have 3 control plane nodes. However, if your cluster i
 
 Scaling a cluster to 4 or 5 control plane nodes is available only on bare metal platforms.
 
+[NOTE]
+====
+Starting in {product-title} 4.18, you can install a cluster with 4 or 5 control plane nodes using the Agent-based Installer or the Assisted Installer.
```

Comment:
The info on line 9 is no longer completely accurate so I think it's reasonable to make a change here to avoid creating inconsistency in the docs.
Is installing with 4/5 control plane nodes really only supported is Agent/Assisted, or has it been supported previously on other baremetal installs and only in 4.18 added to agent/assisted?

Rather than adding a note to correct the inaccuracy it might be better to rewrite the existing text to avoid it in the first place, e.g.

> In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, it may have up to 5 control plane nodes. You can scale up an existing cluster on the bare metal platform with fewer than 5 control plane nodes as a postinstallation task. ...

_assuming_ that actually describes the current situation. (Also, you will no doubt write that last sentence better than I did :slightly_smiling_face:)

Reply:
Sounds good! I asked Bob and he was not aware of any other support prior to ABI/Assisted. I'll remove this note and use your suggested wording to replace the inaccurate text instead, thanks.